app/src/main/java/com/sanaa/tudee_assistant/data/local/entity/CategoryEntity.kt

```diff
@@ -0,0 +1,22 @@
+package com.sanaa.tudee_assistant.data.local.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "categories")
+data class CategoryEntity(
```

Comment:
Let's rename the suffix of the room entity from Entity to LocalDto

```suggestion
data class CategoryLocalDto(
```